### PR TITLE
fix(sandbox): hides org selector for sandbox mode

### PR DESCRIPTION
**Pull request by scefali**
This PR makes sure the org selector is hidden for the sandbox in the sidebar:

Screen Shot 2022-04-25 at 4 33 38 PM

### CODE REVIEW

File: tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+      organization,
+    },
+  ]);
   return render(
     <SidebarDropdown
       orientation="left"
       collapsed={false}
       user={user}
       config={config}
-      organization={organization}
```
Comment (scefali) [inline tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx:16]: note this was a bug before

Comment (Neo-Zhixing) [reply tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx:16]: I wonder why TypeScript was OK about it

**Review by PixelDust22 — APPROVED** [review]

File: tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx (hunk 1 above)
Comment (scefali) [reply tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx:16]: @Neo-Zhixing It's because it's a Javascript file (it's a test)

Comment (Neo-Zhixing) [reply tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx:16]: ok lol i was blind

**Pull request merged**